microsoftgraph/microsoft-graph-explorer-v4
# Fix: Default overlay on collections review panel

## Overview
Closes #2376 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes
The update can be seen on the playwright snapshot diff

## Testing Instructions

* How to test this PR
CLick on the Resources tab
Click on any kebab menu then select Add to collections
Click on Preview collections
Notice the fix
